packages/vouching/contracts/Vouching.sol

```diff
@@ -0,0 +1,101 @@
+pragma solidity ^0.4.24;
+
+
+import "./ZepToken.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract Vouching {
+  using SafeMath for uint256;
+
+  struct Dependency {
```

Comment:
Not a fan of the name. `Dependency` makes sense in a context where it is a dependency of something else. I'd prefer `PackageEntry` or something like that.

Reply:
Also don't like `Dependency`, but `PackageEntry` is somewhat confusing.. I don't have a better proposal though, any other ideas? `Entry` on its own is too dry?